README.md

```diff
@@ -49,6 +49,32 @@ int32_t my_value = data["b"]["a"].value<int32_t>();
 - **No external dependency**: *eProsima xtypes*'s only dependencies are from *std*.
 - **Easy to use**: Compresive API and intuitive concepts.
 
+## Build
+*eprosima xtypes* is header only library, you can simply copy files located in the include folder into your project and include them.
```

Comment:
*eprosima xtypes* is header only library: you can simply copy the files located in the include folder into your project and include them.